nodejs/eks/nodegroup.ts

```diff
@@ -1231,9 +1231,12 @@ ${customUserData}
             metadataOptions: args.metadataOptions,
             userData: userdata,
             tagSpecifications: args.launchTemplateTagSpecifications,
-            monitoring: {
-                enabled: args.enableDetailedMonitoring,
-            },
+            monitoring:
```

Comment:
This changed by pulling the new upstream provider version. Without this we're always getting diffs when the `monitoring` block is empty (i.e. all it's properties are undefined`

Reply:
You can throw an issue in pulumi-aws if relevant happy to dig deep into that.

Reply:
That's one of the permadiffs on empty objects that we've already found some of (empty blocks are not saved to state).

I can cut a ticket to look into it holistically